compile/service.go

```diff
@@ -62,11 +63,22 @@ func compileService(file string, src *ast.Service) (*ServiceSpec, error) {
 		functions[function.Name] = function
 	}
 
+	annotations, err := compileAnnotations(src.Annotations)
+
+	if err != nil {
```

Comment:
Nit: Usually, error checks for things are right next to the statement that created the error. No empty line.